File: app/src/main/java/com/viafourasample/src/activities/article/ArticleViewModel.java
package com.viafourasample.src.activities.article;

import com.viafourasample.src.managers.StoryManager;
import com.viafourasample.src.model.Story;

public class ArticleViewModel {
    private Story story;

    public ArticleViewModel(String containerId) {
        this.story = StoryManager.getInstance().getStory(containerId);
    }

    public Story getStory() {
        return story;
    }
}

File: app/src/main/java/com/viafourasample/src/activities/article/ArticleViewModel.kt
package com.viafourasample.src.activities.article

import com.viafourasample.src.managers.StoryManager
import com.viafourasample.src.model.Story

class ArticleViewModel(containerId: String) {
    val story: Story = StoryManager.getInstance().getStory(containerId)
}
